great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py

```diff
@@ -46,135 +46,108 @@
-                    (otherwise, relative entropy would immediately go to infinity).
-                    * If the partition includes values that are not present in the column, the test will simply
-                    include \
-                    zero weight for that value.
-
-                If the partition_object is continuous, this expectation will discretize the values in the column
-                according \
-                to the bins specified in the partition_object, and apply the test to the resulting distribution.
-
-                    * The internal_weight_holdout and tail_weight_holdout parameters provide a mechanism to weaken the \
-                    expectation, since an expected weight of zero would drive relative entropy to be infinite if any
-                    data \
-                    are observed in that interval.
-                    * If internal_weight_holdout is specified, that value will be distributed equally among any
-                    intervals \
-                    with weight zero in the partition_object.
-                    * If tail_weight_holdout is specified, that value will be appended to the tails of the bins \
-                    ((-Infinity, min(bins)) and (max(bins), Infinity).
-
-              If relative entropy/kl divergence goes to infinity for any of the reasons mentioned above, the observed
-              value\
-              will be set to None. This is because inf, -inf, Nan, are not json serializable and cause some json
-              parsers to\
-              crash when encountered. The python None token will be serialized to null in json.
-
-            See also:
-                :func:`expect_column_chisquare_test_p_value_to_be_greater_than \
-                <great_expectations.dataset.dataset.Dataset.expect_column_unique_value_count_to_be_between>`
-
-                :func:`expect_column_bootstrapped_ks_test_p_value_to_be_greater_than \
-                <great_expectations.dataset.dataset.Dataset.expect_column_unique_value_count_to_be_between>`
-
-            """
+    """Expect the Kulback-Leibler (KL) divergence (relative entropy) of the specified column with respect to the partition object to be lower than the provided threshold.
+
+    KL divergence compares two distributions. The higher the divergence value (relative entropy), the larger \
+    the difference between the two distributions. A relative entropy of zero indicates that the data are \
+    distributed identically, when binned according to the provided partition.
+
+    In many practical contexts, choosing a value between 0.5 and 1 will provide a useful test.
+
+    This expectation works on both categorical and continuous partitions. See notes below for details.
+
+    expect_column_kl_divergence_to_be_less_than is a \
+    [Column Aggregate Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations).
+
```

Comment:
This one looks great, but doesn't seem to be appearing the search results of `Explore Expectations` in our `https://greatexpectations.io/expectations/` site. Could you confirm?

Reply:
That's correct... it's hardcoded to skip for now https://github.com/great-expectations/great_expectations/blob/develop/assets/scripts/build_gallery.py#L298-L303